app/private_publisher.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+
+from app.queue_publisher import QueuePublisher
+
+from cryptography.fernet import Fernet
+
+
+class PrivatePublisher(QueuePublisher):
+
+    @staticmethod
+    def encrypt(message, secret):
+        """
+        Message may be a string or bytes.
+        Secret key must be 32 url-safe base64-encoded bytes.
+
```

Comment:
Should spacing around comments be consistent? There is white space at the end of this comment, but not the next.